python/src/deltachat/__init__.py

```diff
@@ -2,7 +2,12 @@
 from deltachat.capi import ffi
 from deltachat.account import Account  # noqa
 
-__version__ = "0.600.0"
+from pkg_resources import get_distribution, DistributionNotFound
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    # package is not installed
+    pass
```

Comment:
Might be better to provide a valid version anyway?  Objects that are sometimes there and sometimes not are not very nice to work with, even if this is just a local-development-only edge case.